circle.yml

```diff
@@ -459,6 +500,8 @@ jobs:
     docker:
       - image: mbgl/61abee1674:android-ndk-r18
     steps:
+      - generate-core-version-name:
+          version: "1.0.0-SNAPSHOT"
```

Comment:
As mentioned above, this is not needed. We should use the version from `gradle.properties`.

Reply:
Great catch. I also noticed when publishing locally.